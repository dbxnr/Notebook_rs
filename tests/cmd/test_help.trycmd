```
$ nb -h
Notebook 0.2.1

Note taking

USAGE:
    nb [OPTIONS]

OPTIONS:
    -d, --delete <delete>        Delete specific entry
    -e, --edit <edit>...         Edit specific entry
    -h, --help                   Print help information
    -j, --notebook <notebook>    Specify a notebook
    -l, --list [<list>...]       List entries
    -n, --new [<new>...]         Create a new note
    -r, --read <read>            Display specific entry
    -s, --search <search>        Search entries for text
    -v, --verbose                Quantity of information
    -V, --version                Print version information

```