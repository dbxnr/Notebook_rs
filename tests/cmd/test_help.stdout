CLI utility for plaintext notetaking.

Usage: nb [OPTIONS] <COMMAND>

Commands:
  new, -n, --new        Create a new note
  list, -l, --list      List entries
  edit, -e, --edit      Edit specific entry
  read, -r, --read      Display specific entry
  delete, -d, --delete  Delete specific entry
  search, -s, --search  Search entries for text
  help                  Print this message or the help of the given subcommand(s)

Options:
  -j, --notebook <notebook_name>  Specify a notebook name [default: default]
  -v, --verbose...                Quantity of information
  -c, --config <config>           Path of config file to read
  -h, --help                      Print help information
